packages/abi-coder/test/Interface.test.ts

```diff
@@ -766,4 +766,42 @@ describe('Abi interface', () => {
       );
     });
   });
+
+  describe('decodeLog', () => {
+    it('should return decoded log by id', () => {
+      const data = exhaustiveExamplesInterface.decodeLog('0x01000000000000000000000000000020', '0');
```

Comment:
The `logId` passed here should be `"8500535089865083573"` when the sway program is built with `forc 0.59.0`.